docs/SPEC.md

```diff
@@ -182,21 +182,24 @@ When several durations are specified together, larger units must appear before s
 
 Durations represent a length of time.
 Lengths of time are dependent on specific instants in time they occur and as such, durations do not represent a fixed amount of time.
-No amount of seconds is equal to a day, as days vary in their number of seconds.
-No amount of days is equal to a month, as months vary in their number of days.
-A duration consists of three basic time units: seconds, days and months.
+There are no amount of days equal to a month, as months vary in their number of days.
+Durations are a vector of the magnitude of the duration units and the sign of the duration (positive or negative).
 
-Durations can be combined via addition and subtraction.
-Durations can be multiplied by an integer value.
+Durations cannot be combined by addition and subtraction.
+This is because all magnitudes in the vector need to retain the same sign and that cannot be guaranteed when using addition and subtraction.
```

Comment:
I wanted to personally avoid any word that implied what was in the vector mostly because I didn't want to prescribe an implementation. But I agree that it is a bit confusing why the vector cannot have different signs because vector traditionally allow exactly that. So would a pair. Maybe we can call it a tuple? We would say the duration is a sign and a tuple of magnitudes (or positive integers which would include zero).

Reply:
I agree we want to avoid prescribing an implementation if possible. Maybe just say that a duration is made up both month and seconds as separate units and state that the sign is for the entire duration? 

But if that makes it to awkward to describe then I like tuple.